What is w_multi in README?

**Describe the bug**
There is no information about `w_multi` in https://github.com/uber/causalml#interpretable-causal-ml. What is the variable?

The error massage is as below;
```
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-10-e7d0d3c47f9f> in <module>
      8 
      9 slearner = BaseSRegressor(LGBMRegressor(), control_name='control')
---> 10 slearner.estimate_ate(X, treatment, y)
     11 slearner_tau = slearner.fit_predict(X, treatment, y)
     12 

/opt/conda/lib/python3.6/site-packages/causalml/inference/meta/slearner.py in estimate_ate(self, X, treatment, y, return_ci, bootstrap_ci, n_bootstraps, bootstrap_size)
    200             The mean and confidence interval (LB, UB) of the ATE estimate.
    201         """
--> 202         te, yhat_cs, yhat_ts = self.fit_predict(X, treatment, y, return_components=True)
    203 
    204         ate = np.zeros(self.t_groups.shape[0])

/opt/conda/lib/python3.6/site-packages/causalml/inference/meta/slearner.py in fit_predict(self, X, treatment, y, return_ci, n_bootstraps, bootstrap_size, return_components, verbose)
    159                 UB [n_samples, n_treatment]
    160         """
--> 161         self.fit(X, treatment, y)
    162         te = self.predict(X, treatment, y, return_components=return_components)
    163 

/opt/conda/lib/python3.6/site-packages/causalml/inference/meta/slearner.py in fit(self, X, treatment, y)
     76         """
     77         X, treatment, y = convert_pd_to_np(X, treatment, y)
---> 78         check_treatment_vector(treatment, self.control_name)
     79         self.t_groups = np.unique(treatment[treatment != self.control_name])
     80         self.t_groups.sort()

/opt/conda/lib/python3.6/site-packages/causalml/inference/meta/utils.py in check_treatment_vector(treatment, control_name)
     17     if control_name is not None:
     18         assert control_name in treatment, \
---> 19             'Control group level {} not found in treatment vector.'.format(control_name)
     20 
     21 

AssertionError: Control group level control not found in treatment vector.
```

--- a/README.md
+++ b/README.md
@@ -111,6 +111,11 @@ Causal ML provides methods to interpret the treatment effect models trained as f
 
 ```python
 from causalml.inference.meta import BaseSRegressor, BaseTRegressor, BaseXRegressor, BaseRRegressor
+from causalml.dataset.regression import synthetic_data
+
+# Load synthetic data
+y, X, treatment, tau, b, e = synthetic_data(mode=1, n=10000, p=25, sigma=0.5)
+w_multi = np.array(['treatment_A' if x==1 else 'control' for x in treatment]) # customize treatment/control names
 
 slearner = BaseSRegressor(LGBMRegressor(), control_name='control')
 slearner.estimate_ate(X, w_multi, y)

